### DISCUSSION TITLE
GitHub Action Event trigger

### DISCUSSION THREAD

**Discussion by dzervas** [community/community#2959] [Discussions] [answered]
I'd really like to fire an action event when something happens in a discussion, much like with issues - create, edit, comment, etc. Is there such a thing already and I didn't notice?

**Comment by becca** [comment, marked as answer]
Not yet available but is on our roadmap. We'll be working to make this happen after an API and webhooks can be made available. Stay tuned for more information over in our [public roadmap](github.com/github/roadmap)

**Reply by chrisreddington** [reply]
I was going to suggest the exact same, as I've just moved my Podcast's "Episode Suggestions" and "Become a Guest" workflows to GitHub Discussions. A GitHub action workflow trigger would be great - especially if I can have some kind of GitHub discussion template that I can then process as part of an action workflow. Looking forward to this!

PS - Looks like the public roadmap link above is broken. Just in case anyone is looking, [it's here](https://github.com/github/roadmap/projects/1)

**Reply by ahmadchen** [reply]
hi 😵

**Reply by lauracavaliere** [reply]
I would also like to trigger a GitHub Action when something happens in a discussion and I saw from [this](https://github.com/github/feedback/discussions/2943#discussioncomment-605964) link that discussions webhook are available but I can't still use the `discussions` tag since it seems to be a not valid event name.

**Reply by Martii** [reply]
Also noticed that converting a Discussion to an Issue didn't trigger ISSUE_TEMPLATE events yet.
